### ISSUE TITLE
Providing zeal as a snap

### ISSUE THREAD

**Issue by laborg** [zealdocs/zeal#1062]
It would be great if zeal would be available as a snap for ubuntu. An outdated version is available here: https://github.com/casept/snap-zeal/blob/master/snapcraft.yaml

Do you have a plan to offer zeal as a snap?

**Comment by trollixx** [comment]
Yes, adding Snap package is on my to-do list, there's even https://github.com/zealdocs/packaging-snap, which has been kind of abandoned. I believe @jkozera tried to build a fresh package, but faced some strange issues with D-Bus.
